locales/fr.json

```diff
@@ -0,0 +1,95 @@
+{
+  "welcome-back-title": "Bon retour dans Buttercup.",
+  "unlock-archive": "Déverrouillez une archive pour commencer ({os}).",
+  "welcome-title": "Bienvenue dans Buttercup.",
+  "welcome-caption": "Vous n’avez pas encore ajouté d’archive. Pourquoi ne pas en ajouter une ?",
+  "open-archive-file": "Ouvrir un ficier archive",
+  "new-archive-file": "Créer un fichier archive",
+  "connect-cloud-sources": "Connecter des sources sur le Cloud",
+  "add-archive": "Ajouter une archive",
+  "cancel": "Annule",
+  "go-back": "Retour en arrière",
+  "new-archive": "Nouvelle archive",
+  "open-archive": "Open Archive",
+  "open-in-buttercup": "Ouvrir dans Buttercup",
+  "master-password": "Mot de passe principal",
+  "confirm": "Confirmer",
+  "nevermind": "Pas grave…",
+  "password": "Mot de passe",
+  "language": "Langue",
+  "copy-to-clipboard": "Copier dans le presse-papier",
+  "edit": "Éditer",
+  "view": "Visualiser",
+  "window": "Fenêtre",
+  "help": "Aide",
+  "visit-our-website":"Visitez notre site",
```

Comment:
Even though this is fine, maybe 'Visitez notre site web'  would be better and again for consistency I would choose 'Visiter' but it depends on the context of how it is used in the app. For @perry-mitchell : in French you can translate 'Visit' as 'Visitez' which the imperative or 'Visiter' which is the infinitive. On action like 'Confirm' or 'Cancel' we usually use infinitive but for a text on web page that is supposed to invite people to do something, we would use imperative.

Reply:
That makes sense.. I would also expect that the visit call-to-action would be imperative.

Reply:
Done